solv/tdnfpackage.c

```diff
@@ -980,6 +980,7 @@ uint32_t
 SolvCountPkgByName(
     PSolvSack pSack,
     const char* pszName,
+    int nSource,
```

Comment:
can we make this a part of some data structure? or else we will end up passing this function arg everywhere